pkg/apis/core/validation/validation.go

```diff
@@ -5272,6 +5271,12 @@ func ValidateSecurityContext(sc *core.SecurityContext, fldPath *field.Path) fiel
 		}
 	}
 
+	if sc.ProcMount != nil {
+		for _, msg := range IsValidProcMount(*sc.ProcMount) {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("procMount"), *sc.ProcMount, msg))
```

Comment:
use field.NotSupported and provide the valid values

Reply:
done